portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsEventAnalysis.testcase

```diff
@@ -280,6 +280,119 @@ definition {
 		}
 	}
 
+	@description = "Bug ID: LRAC-11659 | Automation ID: LRAC-11726 | Test Summary: Check Default Events Counted as Default when event sent by console"
+	@priority = "5"
+	test CheckDefaultEventsCountedAsDefaultWhenEventSentByConsole {
+		var layoutName = "Custom 1";
+		var globalAttributesList = "canonicalUrl,pageTitle,referrer,url";
+
+		var disableAttributesList = "${globalAttributesList}";
+		var attributesList = "category,like";
+
+		task ("Navigation to default events tab") {
+			ACUtils.launchAC();
+
+			ACNavigation.goToSettings();
+
+			ACSettings.goToDefinitions();
+
+			ACSettings.goToEvents();
+
+			ACNavigation.openItem(itemName = "Default Events");
+		}
+
+		task ("Access the assetViewed") {
+			ACNavigation.openItem(itemName = "assetViewed");
+		}
+
+		task ("Copy the sample JS and replace an existing attribute with a new one") {
+			ACCustomEvents.copySampleJavascript(
+				attributesList = "${attributesList}",
+				disableAttributesList = "${disableAttributesList}");
+		}
+
+		task ("Close sessions") {
+			ACUtils.closeAllSessions();
+
+			ACUtils.launchDXP();
+		}
+
+		task ("Start Har recording") {
+			ProxyUtil.startHarRecording("pageViewed");
+		}
+
+		task ("Send the JavaScript custom event") {
+			var sampleJavascript = "${sampleJavascript}";
```

Comment:
I could be wrong but I don't think you need to declare sameJavascript as a variable here since it is already declared during `ACCustomEvents.copySampleJavascript`